src/window.py

```diff
@@ -291,6 +293,20 @@ def get_next_row():
 
         def get_cur_row():
             return self.cur_row
+        
+        # printer choice
+        printer_label = QLabel(locales.getLocale().PrinterName)
+
+        printer_basename = ""
+        try:
+            printer_basename = path.basename(sett().hardware.printer_dir)
+        except:
+            # set default path to printer config
+            sett().hardware.printer_dir = ""
```

Comment:
I think there is no need, by default when we didn't choose any directory it will be just `data/calibration_data.csv`. If user wants to use calibration, he will copy to some directory